app/services/FinancialTransactionsService.scala

```diff
@@ -18,32 +18,31 @@ package services
 
 import javax.inject.{Inject, Singleton}
 import audit.AuditingService
-import audit.models.{DirectDebitCheckRequestAuditModel, DirectDebitsCheckResponseAuditModel,
-  FinancialTransactionsRequestAuditModel, FinancialTransactionsResponseAuditModel}
+import audit.models.{DirectDebitCheckRequestAuditModel, DirectDebitsCheckResponseAuditModel, FinancialTransactionsRequestAuditModel, FinancialTransactionsResponseAuditModel}
```

Comment:
This line is generating a little scalastyle warning because of excessive line length. Can you just use an underscore here?